src/copyFile.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+
+function copyFile(src, dest) {
+  if (src === dest) {
+    console.log('You are trying to copy to the same location');
+
+    return;
+  }
+
+  try {
+    fs.copyFileSync(src, dest);
```

Comment:
pls try achieving the task using `readFileSync` and `writeFileSync` instead of `copyFileSync`